class.ext_update.php

```diff
@@ -7,6 +7,12 @@
  * @package TYPO3_Aimeos
  */
 
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Core\Database\ConnectionPool;
```

Comment:
Wasn't `ConnectionPool` available for TYPO3 8+ only?

Reply:
Now you have my even fuller attention.

I just ran `use`ing a none-existing-class-setup through our debugger test scenario. No warning, notice or even a triggering of an autoloader. 
(5.5 till 7.2)

How can I reproduce that fatal?

Reply:
Seems like I was wrong. This works:
```
use Non\Existing\Classname;
class Test {}
```
And this fails as expected:
```
use Non\Existing\Classname;
class Test extends Classname {}
```
The `use` statement for non-existing classes doesn't seem to generate any errors but I wouldn't rely on that "feature" in the future ...

Reply:
> wouldn't rely on that "feature" in the future
True.